src/clj/web/stats.clj

```diff
@@ -113,11 +114,11 @@
   (doseq [player original-players]
     (if (:side player)
       (inc-game-stats db (get-in player [:user :_id]) (game-record-start player))
-      (println "NULL start player side in stats for gameid" gameid)))
+      (timbre/error (str "NULL start player side in stats for gameid" gameid))))
   (doseq [player ending-players]
     (if (:side player)
       (inc-game-stats db (get-in player [:user :_id]) (game-record-end state player))
-      (println "NULL end player side in stats for gameid" gameid))))
+      (timbre/error (str "NULL end player side in stats for gameid" gameid)))))
```

Comment:
this and the others will result in "NULL end player ... for gameid12345" because `println` adds spaces between args and `str` doesn't.